Makefile

```diff
@@ -333,6 +333,15 @@ clippy_integer: install_rs_check_toolchain
 		--features=$(TARGET_ARCH_FEATURE),integer,experimental \
 		-p $(TFHE_SPEC) -- --no-deps -D warnings
 
+.PHONY: clippy_zk # Run clippy lints on TFHE-rs with the zk-pok feature enabled
+clippy_zk: install_rs_check_toolchain
+	RUSTFLAGS="$(RUSTFLAGS)" cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" clippy \
+		--features=$(TARGET_ARCH_FEATURE),zk-pok \
+		-p $(TFHE_SPEC) -- --no-deps -D warnings
+	RUSTFLAGS="$(RUSTFLAGS)" cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" clippy \
+		--features=$(TARGET_ARCH_FEATURE),zk-pok,shortint \
+		-p $(TFHE_SPEC) -- --no-deps -D warnings
+
```

Comment:
check clippy_zk_pok maybe ? or choose a name a bit more different

Reply:
clippy_zk_pok is already used for clippy on the tfhe-zk-pok crate...